apps/crn-server/src/data-providers/reminders.data-provider.ts

```diff
@@ -158,6 +213,40 @@ const getUserTeamIds = (
   return userTeams;
 };
 
+const getUserWorkingGroupIds = (
+  workingGroups: Maybe<
```

Comment:
How can I avoid having this type here ?

Reply:
I think this type is equivalent to
```
type ReferencingWorkingGroupsContents = NonNullable<
  FetchReminderDataQuery['findUsersContent']
>['referencingWorkingGroupsContents'];
```

Reply:
look above, probably something like `NonNullable<FetchReminderDataQuery['findUsersContent']>['referencingWorkingGroupsContents']`